examples/node-agent-live/index.js

```diff
@@ -5,6 +5,36 @@ const { join } = require("path");
 
 const deepgram = createClient(process.env.DEEPGRAM_API_KEY);
 
+// Add WAV header generation function
```

Comment:
@naomi-lgbt  I fixed this example as the wav out files were not playable as there wasn't any wav file headers. this change fixes that issue so now users can play the audio from the chat logs. 

This passed my tests otherwise.